packages/runtime/src/helpers/files.ts

```diff
@@ -271,13 +272,16 @@ export const moveStaticPages = async ({
   }
 
   if (shortRevalidateRoutes.length !== 0) {
-    console.log(outdent`
+    logger.info(outdent`
       The following routes use "revalidate" values of under ${MINIMUM_REVALIDATE_SECONDS} seconds, which is not supported.
       They will use a revalidate time of ${MINIMUM_REVALIDATE_SECONDS} seconds instead.
     `)
+
+    // Disabling lint rule as the logger doesn't have a table formatter
+    // eslint-disable-next-line no-console
     console.table(shortRevalidateRoutes)
```

Comment:
I wasn't sure what we wanted to do here. We'd have to write some code to format as a table so I've left `console.table` as it was.